### PR TITLE
Update translation requests contact

### PR DESCRIPTION
**Pull request by NimaBoscarino**
# What does this PR do?
Updates the contact for translation requests to GuggerSylvain (@sgugger - if you're alright with that!)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?
Documentation: @sgugger, @osanseviero

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Sure why not. Thanks for updating!

**Review by osanseviero — APPROVED** [review]
Thanks a lot! Not a fan of people tagging in Twitter but it should be alright

File: docs/TRANSLATING.md [hunk 1]
```diff
@@ -54,5 +54,5 @@ The fields you should add are `local` (with the name of the file containing the
 
 Once you have translated the `_toctree.yml` file, you can start translating the [MDX](https://mdxjs.com/) files associated with your docs chapter.
 
-> 🙋 If you'd like others to help you with the translation, you can either [open an issue](https://github.com/huggingface/transformers/issues) or tag @[espejelomar](https://twitter.com/espejelomar)
- on Twitter to gain some visibility.
\ No newline at end of file
+> 🙋 If you'd like others to help you with the translation, you can either [open an issue](https://github.com/huggingface/transformers/issues) or tag @[GuggerSylvain](https://twitter.com/GuggerSylvain)
+ on Twitter to gain some visibility.
```
Comment (sgugger) [inline docs/TRANSLATING.md:58]:
```suggestion
> 🙋 If you'd like others to help you with the translation, you should [open an issue](https://github.com/huggingface/transformers/issues) and tag @sgugger.
```

**Review by sgugger — COMMENTED** [review]
Somehow missed that. Absolutely don't want to be tagged on Twitter for this 😅

**Review by sgugger — APPROVED** [review]
Thanks for flagging @osanseviero !

**Comment by NimaBoscarino** [comment]
Doesn't look like I have merge privileges on this repo, could you merge it @sgugger?

**Comment by sgugger** [comment]
Just waiting for the last green tick and will do so!

**Pull request merged**